docs/development.md

```diff
@@ -322,6 +322,7 @@ You can also customize the configuration of the CAPZ cluster (assuming that `SKI
 [image_pull_secrets]: https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod
 [gettext]: https://www.gnu.org/software/gettext/
 [gettextwindows]: https://mlocati.github.io/articles/gettext-iconv-windows.html
+[go.mod]: https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/go.mod#L29
```

Comment:
That line will likely change in the future. Perhaps, just a link to the go.mod file.